Resources/Prototypes/Catalog/Fills/Lockers/suit_storage.yml

```
@@ -319,3 +319,26 @@
+  - type: StorageFill
+    contents:
+        - id: OxygenTankFilled
+        - id: ClothingOuterHardsuitShanlinUnpainted
+        - id: ClothingMaskBreath
+
+- type: entity
+  id: SuitStorageShiwei
+  parent: SuitStorageBase
+  suffix: CSA-54UA
+  components:
+  - type: StorageFill
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 2

(indentation)